### DISCUSSION TITLE
Removing Eskild as a core team member as he transitions into a new season.

### DISCUSSION THREAD

**Discussion by dharhas** [nebari-dev/nebari#2160] [Community]
@iameskild is moving on to a new role and does not expect to be contributing heavily to Nebari in the near term. Per his request we are removing him as a core team member. 

We thank him for all his contributions and his service as the release manager for Nebari for I think over a year now. It has been a pleasure working with you Eskild.

**Comment by trallard** [comment]
Added @iameskild to our [Emeritus sustainers team](https://github.com/orgs/nebari-dev/teams/emeritus-sustainers/members).

🙏🏽 Thank you Eskild for all your valuable contributions and hard work throughout your tenure at nebari
